src/components/screens/ProcessScreen.jsx

```diff
@@ -24,11 +25,24 @@ const ProcessScreen = () => {
           dispatch(
             addContentfulSupplier({
               contentfulId: s.sys.id,
-              id: s.fields.supplierId["en-GB"].toString(),
+              id: parseInt(s.fields.supplierId["en-GB"], 10),
               name: s.fields.name["en-GB"],
               dataAvailable: s.fields.dataAvailable["en-GB"],
             }),
           );
+
+          if (
+            s.fields.whitelabelSupplier &&
+            s.fields.whitelabelSupplier["en-GB"]
+          ) {
+            dispatch(
+              setWhitelabelSupplierId({
+                id: s.fields.supplierId["en-GB"],
```

Comment:
does it matter that above this is turned into an int, but this is not? (just want to make sure we don't end up in type confusion where sometimes the id is a string and others an int)